### DISCUSSION TITLE
Is git-filter-repo still active and supported?

### DISCUSSION THREAD

**Discussion by gdlxn-ibm** [newren/git-filter-repo#570] [General]
Is git-filter-repo still active and supported?  The reason I ask is that there hasn't been an update to the `git-filter-repo` script since Oct 9, 2022 and the last update to the repository was on Apr 12, 2023.

**Comment by newren** [comment]
It's a fair question.  My former employer started ramping down my git contributions in early 2022 (actually back in 2021 but it was phrased as "temporary" at the time), leaving me to pick and choose which git projects got shortchanged even though I tried to keep as much going as I could in my spare time.  But 2023 was particularly rough, with three family members getting some rough news and all needing a fair amount of support -- combined with my employer telling me multiple times through multiple people each that it was "time to apply my talents to things other than git", i.e. no more support for git stuff, plus I was put on a new team and needed to learn a number of things to ramp up and be productive there.  Between it all, I just couldn't keep it up, and worse -- I was burnt out from attempting to overdo it in my spare time for longer than was wise.  I guess I probably should have let folks know, but I really had no idea that my attempts on multiple fronts to change various aspects about my situation so I could do more kept taking much longer than I expected.

However, 2 of the 3 family members are doing better this year.  And I started a new job a couple weeks ago, and my new employer is supportive of letting me spend work time on git (and git-filter-repo) again.  There's a lot for me to catch up on, but you can expect to see git-filter-repo changes and a new release again soon.

**Comment by gdlxn-ibm** [comment]
Elijah, thanks for the quick response.  I'm sorry to hear about the troubles you've had over the last couple of years.  I'm glad to hear that your new employer supports your work on git and git-filter.

My employer is supportive of open source work; they actually actively encourage open source work.  I'd like to work on Issue #401 and the new Issue #569 I opened earlier today. Any suggestions to help me get started on these issues would be greatly appreciated.

**Comment by newren** [comment]
Given the answer above (and my comments on the two linked tickets which can be tracked separately), plus the new release last month and more pending changes, I'm going to close this out and consider it answered.  Thanks, everyone.
